tests/e2e/system_test/test_node_kernel_crash_ceph_fsync.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import random
+
+from time import sleep
+from ocs_ci.ocs import constants, node
+from ocs_ci.framework.testlib import E2ETest, tier1, bugzilla, polarion_id
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2075068")
+@polarion_id("OCS-3947")
+class TestKernelCrash(E2ETest):
+    """
+    Tests to verify kernel crash
+    """
+
+    pvc_size = 1
+    original_dir = "/var/lib/www/html/"
+    result_dir = "mydir"
+
+    def creates_files(self, pod_obj):
+        while True:
+            for i in range(1, 125):
+                pod_obj.exec_cmd_on_pod(
+                    f"dd if=/dev/zero of={self.original_dir}{self.result_dir}/emp.txt bs=1M count=1"
+                )
+
+    def remove_files(self, pod_obj):
+        while True:
+            for i in range(1, 125):
+                pod_obj.exec_cmd_on_pod(
+                    f"rm {self.original_dir}{self.result_dir}/emp.txt"
+                )
+
+    def test_node_kernel_crash_ceph_fsync(self, pvc_factory, teardown_factory):
+        """
+        1. Create 1GiB PVC
+        2. Attach PVC to an application pod
+        3. Copy files Files_create_delete.py and fsync.py to pod
+        4. Execute parallel Files_create_delete.py and fsync.py
+        5. Check Node gets Panic or not
+        """
+
+        worker_nodes_list = get_worker_nodes()
+
+        # Create a Cephfs PVC
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            status=constants.STATUS_BOUND,
+        )
+
+        # Confirm PVCs are Bound
+        log.info("Verifying the CephFS PVC are Bound")
+        helpers.wait_for_resource_state(
+            resource=pvc_obj, state=constants.STATUS_BOUND, timeout=240
```

Comment:
why not using the default timeout of 60. Is it naccesary?
Also, sine you got the pvc status "bound" in pvc_factory(), it will wait for it.
This way you're waiting twice